lib/python/docs/notebooks/experiment_tracking_demo.ipynb

```diff
@@ -270,7 +263,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "experiment_mlflow.publish(mc_loc=\"performance.performanceMetrics\", run_id=\"\")"
+    "experiment_mlflow.publish(mc_loc=\"performance.performanceMetrics\", run_id=\"bcee7ec632894ca4a4208f105e7d9755\", semver=\"0.2.0\")"
```

Comment:
Same with this could we redact this id and and use a screenshot?

Reply:
Done